DOC-2852-added Token Bank Exception (#292)
tigergraph/gsql-docs: modules/ddl-and-loading/pages/managing-loading-job.adoc

| Invalid primary id
| The number of invalid objects caused by wrong data format for the PRIMARY_ID type

| Incorrect fixed binary length
| The number of invalid objects caused by the mismatch of the length of the data to the type defined in the schema

| Token Bank Exception

> | Exception thrown by functions within the token bank. This is a general error category for failures in user-defined functions and does not provide specific details about the cause.

Reviewer: ```suggestion
| The number of exceptions thrown by user-defined token functions.
```
